docs/community/meetings/2024-05-09.md

```diff
@@ -0,0 +1,201 @@
+# 9 May 2024
+
+## Presentation by T.K. Le
+
+In this meeting [T.K. Le](https://github.com/thaopletk) gave a presentation about a series of COVID-19 modelling projects and how their experiences were impacted by the choice of programming languages, model structure, editors, tools, etc.
+
+Attendance: 4 in person, 4 online.
+
+!!! info
+
+    We welcome presentations about research projects and experiences that relate in any way to reproducibility and good computational research practices.
+    Presentations can be short, informal, and free-form.
+
+    Please contact us if you have anything you might like to present!
+
+## Three projects, four models
+
+This work began in 2022, and was based on code that was originally written by [Eamon](https://github.com/EamonConway) and Camelia.
+T.K. adapted the code to run new scenarios across three separate modelling projects:
+
+1. [Modelling the impact of hybrid immunity on futute COVID-19 waves](https://doi.org/10.1186/s12879-024-09282-4);
+2. [Cost-effective boosting allocations in the post-Omicron era of COVID-19 management](https://doi.org/10.1101/2023.11.14.23298536); and
+3. Confidential scenario modelling for the Australian Department of Health & Aged Care.
+
+The workflow was divided into a series of distinct models:
+
+1. An immunological model written in R and [greta](https://greta-stats.org/);
+
+2. An agent-based model of population transmission written in C++ and R;
```

Comment:
written in C++ (with post-processing in R)

Reply:
Done